Frameworks/Foundation/NSURLConnection.mm

```diff
@@ -54,27 +55,32 @@ @implementation _NSURLConnection_DataDelegate {
 
     StrongId<NSURLResponse> _response;
     StrongId<NSMutableData> _data;
+
+    bool _started;
```

Comment:
Wait, this doesn't even seem like it gets used in the body of this data delegate... but _started gets used in NSURLConnection. Did some wires get crossed?